agents/trials.js

```diff
@@ -57,8 +57,13 @@ function searchTrials(query, page, perPage, filters) {
     .then((response) => response.obj);
 }
 
+function searchTrialsByEntity(entityType, entityName) {
+  return searchTrials(`${entityType}:${entityName}`, 1, 10);
```

Comment:
This will break if the `entityName` has whitespaces. You must escape `entityName` (make sure there're no unescaped `"` in it) and use `${entityType}:"${entityName}"`. This isn't enough to be sure it handles all cases, but should do for now. I've created a new issue to do it later: https://github.com/opentrials/opentrials/issues/172